.github/workflows/maths-ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Maths CI Build
+
+on:
+  push:
+    paths:
+      - 'src/Maths/**/*'
+  pull_request:
+    paths:
+      - 'src/Maths/**/*'
+
+jobs:
+  Test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 5.0.100-preview.7.20366.6
+    - name: Run Tests and Export NUnit
+      run: dotnet test ./src/Maths/Silk.NET.Maths.Tests/Silk.NET.Maths.Tests.fsproj -c Release -f net5.0 -l GitHubActions
+  Coverage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with: 
+        dotnet-version: 5.0.100-preview.7.20366.6
+    - name: Collect coverage
+      run: dotnet test ./src/Maths/Silk.NET.Maths.Tests/Silk.NET.Maths.Tests.fsproj --configuration Release /p:AltCover=true /p:AltCoverCobertura="cobertura.xml"
```

Comment:
dotnet test is done as part of the main CI build pipeline, so could you just append the codecov routine to the existing CI pipeline?

Reply:
not really, as this is using altcover, so those parameters have to be given.

Reply:
Hmm ok, this seems fine for now.